frontend/src/components/pages/CampCreation/ScheduleSessions/SessionSidePanel.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Box } from "@chakra-ui/react";
+import { CreateCampSession } from "../../../../types/CampsTypes";
+import AddSessionsForm from "./AddSessionsForm";
+import CurrentSessionsView from "./CurrentSessionsView";
+
+type SessionSidePanelProps = {
+  scheduledSessions: CreateCampSession[];
+  setScheduledSessions: (sessions: Array<CreateCampSession>) => void;
+};
+
+const SessionSidePanel: React.FC<SessionSidePanelProps> = ({
+  scheduledSessions,
+  setScheduledSessions,
+}: SessionSidePanelProps): React.ReactElement => {
+  const [showAddSessionView, setShowAddSessionView] = React.useState(false);
+
+  return (
+    <Box
+      minWidth="35vw"
+      minHeight="100vh"
+      position="absolute"
+      right="0"
```

Comment:
is this not the default?

Reply:
i think left="0" would be default - w/out this it won't be on the right